shell/exec.go

```diff
@@ -32,6 +32,8 @@ var (
 
 // MenderShellMessage represents a message between the device and the backend
 type MenderShellMessage struct {
```

Comment:
This is starting to look suspiciously like `ws.ProtoMsg` :see_no_evil:

Reply:
It predates ProtoMsg; we need to refactor the code to get rid of this struct, but as it is heavily work in progress, we are delaying it. I'll give it a try...

Reply:
Here we get rid of it, in favour of ProtoMsg: https://github.com/mendersoftware/mender-connect/pull/7/commits/a33a9b70bd9b55586ea5c3ab97c5c4ff1d1d3a93